### PR TITLE
[TT-1209] use the same CL node config in k8s as in docker

### PR DESCRIPTION
**Pull request by Tofel**
We just read it from TOML instead of using hardcoded ones. I also split v1/v2 ocr and forwarder tests into separate ones so that we can read correct TOML config for them. Makefile commands were modified and command names changed!

Tests... tested:
* automation load -> simulated
* keeper benchmark -> simulated
* ocr soak -> simulated
* ocr2 soak -> simulated
* forwarder ocr soak -> simulated
* forwarder ocr2 soak -> simulated

### CODE REVIEW

File: integration-tests/load/automationv2_1/automationv2_1_test.go [hunk 1]
```diff
@@ -292,21 +281,16 @@ Load Config:
 	numberOfUpkeeps := *loadedTestConfig.Automation.General.NumberOfNodes
 
 	for i := 0; i < numberOfUpkeeps+1; i++ { // +1 for the OCR boot node
-		var nodeTOML string
-		if i == 1 || i == 3 {
-			nodeTOML = fmt.Sprintf("%s\n\n[Log]\nLevel = \"%s\"", baseTOML, *loadedTestConfig.Automation.General.ChainlinkNodeLogLevel)
-		} else {
-			nodeTOML = fmt.Sprintf("%s\n\n[Log]\nLevel = \"info\"", baseTOML)
-		}
-		nodeTOML = networks.AddNetworksConfig(nodeTOML, loadedTestConfig.Pyroscope, testNetwork)
```
Comment (anirudhwarrier) [inline integration-tests/load/automationv2_1/automationv2_1_test.go:301]: Would a test with `pyroscope` enabled work if without this line?
Running a test https://github.com/smartcontractkit/chainlink/actions/runs/9644338945/job/26596224343 to confirm

Comment (anirudhwarrier) [reply integration-tests/load/automationv2_1/automationv2_1_test.go:301]: confirmed it works both when enabled and disabled.

File: integration-tests/reorg/automation_reorg_test.go [hunk 2]
```diff
@@ -104,6 +84,30 @@ LimitDefault = 5_000_000
  * Upkeeps are expected to be performed during the reorg.
  */
 func TestAutomationReorg(t *testing.T) {
+	c, err := tc.GetConfig("Reorg", tc.Automation)
```
Comment (anirudhwarrier) [inline integration-tests/reorg/automation_reorg_test.go:87]: Testing if reorg test still works - https://github.com/smartcontractkit/chainlink/actions/runs/9644671951

**Review by anirudhwarrier — APPROVED** [review]
Verified that all Automation tests work as before without any changes.

**Review by chudilka1 — APPROVED** [review]
Does it require updating README?

**Pull request merged**